### PR TITLE
Fix agility pyramid laps

### PR DESCRIPTION
**Pull request by Nightfirecat**
Fixes runelite/runelite#4075

Ref: runelite/runelite#3911 (for visibility changes)

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
 	private final AgilityConfig config;
 
 	@Inject
-	public AgilityOverlay(Client client, AgilityPlugin plugin, AgilityConfig config)
+	AgilityOverlay(Client client, AgilityPlugin plugin, AgilityConfig config)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java]: This should be private

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java [hunk 2]
```diff
@@ -132,7 +132,7 @@ protected void shutDown() throws Exception
 	}
 
 	@Subscribe
-	public void onGameStateChange(GameStateChanged event)
+	void onGameStateChange(GameStateChanged event)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java]: for subscribers we decided to use public for consistency (and also kind of makes sense as it is indirectly called as public)

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java (hunk 1 above)
Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityOverlay.java]: Privated. :+1:

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java (hunk 2 above)
Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java]: No problem, reverted the subscriber visibility changes.

**Pull request merged**